### PR TITLE
fix(vpc):better rendering of subnet in private network

### LINKED ISSUE #3322: Improve rendering of Subnets of private networks in the get cluster command
```
$ scw k8s cluster create
...

Private Network:
ID                   6c1d336b-06e0-4aca-97a3-a7286d13f7f1
Name                 pn-recursing-merkle
OrganizationID       951df375-e094-4d26-97c1-ba548eeb9c42
ProjectID            951df375-e094-4d26-97c1-ba548eeb9c42
Region               fr-par
Tags.0               created-along-with-k8s-cluster
Tags.1               created-by-cli
CreatedAt            2 seconds ago
UpdatedAt            2 seconds ago
Subnets.0.ID         d091ccd0-d334-4922-9e93-519ee6c23a74
Subnets.0.CreatedAt  2 seconds ago
Subnets.0.UpdatedAt  2 seconds ago
Subnets.0.Subnet     172.16.12.0/22
Subnets.1.ID         af62715c-7d83-4696-8041-c090b3d25115
Subnets.1.CreatedAt  2 seconds ago
Subnets.1.UpdatedAt  2 seconds ago
Subnets.1.Subnet     fd49:821a:a311:7296::/64
VpcID                4c837446-e418-4326-af43-4a445ea0aab8
DHCPEnabled          true
```

### PR DESCRIPTION
**Pull request by jremy42**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request.
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Release note for [CHANGELOG](https://github.com/scaleway/scaleway-cli/blob/master/CHANGELOG.md):

```release-note

```

close #3322

### CODE REVIEW

File: internal/namespaces/vpc/v2/custom.go [hunk 1]
```diff
@@ -10,5 +12,10 @@ func GetCommands() *core.Commands {
 	cmds.Remove("vpc", "post")
 	cmds.MustFind("vpc", "private-network", "get").Override(privateNetworkGetBuilder)
 
+	//human.RegisterMarshalerFunc(k8s.NodeStatus(""), human.EnumMarshalFunc(nodeStatusMarshalSpecs))
```
Comment (remyleone) [inline internal/namespaces/vpc/v2/custom.go:15]: Why is this line commented?

Comment (jremy42) [reply internal/namespaces/vpc/v2/custom.go]: oops, a line I forgot to delete

**Comment by remyleone** [comment]
```
$ go run ./cmd/scw vpc private-network list
---------------------------------------------------------------------------------------
An error occurred, we are sorry, please consider opening a ticket on github using: 'scw feedback bug'
Give us as many details as possible so we can reproduce the error and fix it.
---------------------------------------------------------------------------------------
runtime error: invalid memory address or nil pointer dereference
stacktrace from panic: 
goroutine 1 [running]:
runtime/debug.Stack()
        /opt/homebrew/opt/go/libexec/src/runtime/debug/stack.go:24 +0x64
main.cleanup(0x14000237340)
        /Users/rleone/workspace/scaleway-cli/cmd/scw/main.go:42 +0xb4
panic({0x1018ccfe0?, 0x1026b63f0?})
        /opt/homebrew/opt/go/libexec/src/runtime/panic.go:914 +0x218
github.com/scaleway/scaleway-cli/v2/internal/namespaces/vpc/v2.privateNetworkSubnetMarshalerFunc({0x10181e000?, 0x14000633140?}, 0x0)
        /Users/rleone/workspace/scaleway-cli/internal/namespaces/vpc/v2/custom_private_network.go:24 +0x1cc
github.com/scaleway/scaleway-cli/v2/internal/human.marshalInlineSlice({0x10181e000, 0x14000633140, 0x97})
        /Users/rleone/workspace/scaleway-cli/internal/human/marshal.go:359 +0x250
github.com/scaleway/scaleway-cli/v2/internal/human.marshalSlice({0x10181e240?, 0x140006330c8?, 0x1018e98e0?}, 0x14000102e10)
        /Users/rleone/workspace/scaleway-cli/internal/human/marshal.go:326 +0x6d0
github.com/scaleway/scaleway-cli/v2/internal/human.Marshal({0x10181e240, 0x140006330c8}, 0x140006330c8?)
        /Users/rleone/workspace/scaleway-cli/internal/human/marshal.go:81 +0x2fc
github.com/scaleway/scaleway-cli/v2/internal/core.(*Printer).printHuman(0x14000100d20, {0x10181e240, 0x140006330c8}, 0x0)
        /Users/rleone/workspace/scaleway-cli/internal/core/printer.go:202 +0x15c
github.com/scaleway/scaleway-cli/v2/internal/core.(*Printer).Print(0x14000100d20, {0x10181e240?, 0x140006330c8?}, 0x10149e50a?)
        /Users/rleone/workspace/scaleway-cli/internal/core/printer.go:159 +0x12c
github.com/scaleway/scaleway-cli/v2/internal/core.Bootstrap(0x14000713e70)
        /Users/rleone/workspace/scaleway-cli/internal/core/bootstrap.go:281 +0x11c4
main.main()
        /Users/rleone/workspace/scaleway-cli/cmd/scw/main.go:74 +0x314

```

**Comment by Codelax** [comment]
`scw vpc private-network list` does not crash anymore but the rendering is still off:
```
❯ scw vpc pn list -owide
ID                                    NAME             ORGANIZATION ID                       PROJECT ID                            REGION  TAGS                                             CREATED AT   UPDATED AT   SUBNETS  VPC ID  DHCP ENABLED
eafcee16-4658-4690-84d6-ad2d8d576cc4  pn-happy-turing  ee7bd9e1-9cbd-4724-b2f4-19e50f3cf38b  ee7bd9e1-9cbd-4724-b2f4-19e50f3cf38b  fr-par  [created-along-with-k8s-cluster created-by-cli]  3 weeks ago  3 weeks ago  ID                                    Subnet                    Created at   Updated at
6ed2866b-5224-4e98-a22f-12154b9726fd  172.16.16.0/22            3 weeks ago  3 weeks ago
e31b3ba1-6bfb-4245-b1ca-fcf5fd738e55  fd4a:c18e:554d:bc85::/64  3 weeks ago  3 weeks ago  53e50f63-f6c0-44f7-8576-d6edcb965d9c  true
23e11465-6939-4c30-a14e-761beac5740b                                                      pn-thirsty-diffie                     ee7bd9e1-9cbd-4724-b2f4-19e50f3cf38b  ee7bd9e1-9cbd-4724-b2f4-19e50f3cf38b  fr-par  [created-along-with-k8s-cluster created-by-cli]  1 week ago  1 week ago  ID                                    Subnet                    Created at  Updated at
56a8174c-a7c7-494e-bb87-341dd173a056  172.16.8.0/22             1 week ago  1 week ago
c8fdb28f-63a8-44bf-b59d-52b89c7c3a9e  fd4a:c18e:554d:208a::/64  1 week ago  1 week ago  53e50f63-f6c0-44f7-8576-d6edcb965d9c  true


```

**Comment by jremy42** [comment]
fix !

File: internal/namespaces/vpc/v2/custom_private_network.go [hunk 2]
```diff
@@ -136,6 +159,95 @@ func privateNetworkGetBuilder(c *core.Command) *core.Command {
 	return c
 }
 
+//func privateNetworkListBuilder() *core.Command {
```
Comment (remyleone) [inline internal/namespaces/vpc/v2/custom_private_network.go:162]: Why is this code commented?

**Review by remyleone — APPROVED** [review]

**Pull request merged**